rllib/core/models/tf/encoder.py

```diff
@@ -69,5 +69,5 @@ class TfActorCriticEncoder(TfModel, ActorCriticEncoder):
     framework = "tf"
 
     def __init__(self, config: ModelConfig) -> None:
-        ActorCriticEncoder.__init__(self, config)
         TfModel.__init__(self, config)
+        ActorCriticEncoder.__init__(self, config)
```

Comment:
was the order important? if so, can we have a comment on why and how it is important?

Reply:
Yes, it's important, because tf.keras.Model requires you to call it's constructor before other stuff (like creating models) happens.

Reply:
I'm adding a comment

Reply:
oh like nn.Modules in torch. got it. makes sense.